CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## [0.18.1] - 2019-05-31
+
+### Added
+
+- ([GH-505](https://github.com/lingua-pupuli/puppet-vscode/issues/505)) Add hide `PDK New Module` titlebar button
+- ([GH-507](https://github.com/lingua-pupuli/puppet-vscode/issues/507))([GH-508](https://github.com/lingua-pupuli/puppet-vscode/issues/508)) Document Puppet Version Support
```

Comment:
Missing 522

Reply:
~~i.e. updates to editor services and editor syntax~~

OH I see, you added those explicitly into the Fixed section